cleaner/environment_matrix/classes/clean.php

```diff
@@ -83,6 +85,9 @@ static public function execute() {
                 // Obtain the data for this environment only.
                 $matrixdata = local\matrix::get_matrix_data($environment);
 
+                // Set Admin User for admin_write_settings perms
+                \core\session\manager::set_user(get_admin());
```

Comment:
just wondering if this should actually be here, or right back in the cli script like the upgrade script?

Reply:
I Put it here to minimise any potential side effects of having it (should be none), just incase there is something im not seeing that may be affected by the user that is logged in